.github/workflows/presentation.yaml

```diff
@@ -0,0 +1,31 @@
+name: Presentation publish
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - presentation/**
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.1
+
+      - name: Use pnpm
+        uses: pnpm/action-setup@v2.0.1
+        with:
+          version: 6.28.0
+
+      - name: Install and Build 🔧
+        run: |
+          pnpm install
+          pnpm build:presentation
+
+      - name: Deploy to Pages 🚀
```

Comment:
question: Why not to Cloudflare Pages? That's where we wanted to deploy the app, maybe we could deploy the presentation there too?